root_locals.tf

```diff
@@ -60,6 +60,8 @@ locals {
 
   ip_allowlist = concat(local.developer_ip_list, local.trusted_ip_list)
 
+  ip_blocked_list = module.tdr_configuration.terraform_config["ip_blocked_list"]
```

Comment:
Bit confused. From this PR: https://github.com/nationalarchives/da-terraform-configurations/pull/47 these are in the `da-terraform_configurations` not the `tdr_configuration`.

Reply:
The source for tdr_configuration is da-terraform-configurations
```
module "tdr_configuration" {
  source  = "./da-terraform-configurations"
  project = "tdr"
}
```

Reply:
Didn't notice that.